pkgs/by-name/re/resumed/package.nix

```
@@ -18,11 +21,21 @@ buildNpmPackage (finalAttrs: {
 
   npmDepsHash = "sha256-UElS1pEzPv0FnvMGCnqEFBi7JzE8QWRFynkAPHy35FY=";
 
+  postPatch = ''
+    ${lib.getExe jq} '.version = "${finalAttrs.version}"' package.json | ${lib.getExe' moreutils "sponge"} package.json
```

Meh, why not just do a violent `sed` substitution? If someone uses `"` in `version`, they can fix it in their override.

```suggestion
    sed -i 's/"version": ".*"/"version": "${finalAttrs.version}"/' package.json
```

---

Fine for me. I just found the `jq` snippet somewhere and it seemd to be the more ideal tool for the job.

---

> If someone uses `"` in `version`, they can fix it in their override.

sed's wildcards are greedy by default, they already reach the farthest limit, so even values with `\"` will be safe:
```console
$ echo '"oi\"there"' | sed 's/".*"/"shite"/'
"shite"
```

---

I meant the package's `version` attribute: you'd end up with malformed JSON.